pwiz_tools/Skyline/TestUtil/PythonTestUtil.cs

```diff
@@ -185,15 +204,10 @@ public bool InstallPython(BuildLibraryDlg buildLibraryDlg)
             return false;
         }
 
-        protected override void DoTest()
-        {
-            throw new NotImplementedException();
-        }
-
         private void RunNvidiaDialog(MultiButtonMsgDlg nvidiaDlg)
         {
             nvidiaDlg.ClickNo();
-            MessageDlg confirmDlg = WaitForOpenForm<MessageDlg>(600000);
+            MessageDlg confirmDlg = AbstractFunctionalTest.WaitForOpenForm<MessageDlg>(600000);
```

Comment:
You should use the change I supplied privately:

`           // We aren't running with admin privilege so we have to say no to the offer to install nvidia libraries
            var confirmDlg = AbstractFunctionalTest.ShowDialog<MessageDlg>(nvidiaDlg.ClickNo, 600000);
`

As it is, you're running UI code on the test thread, which causes intermittent failures